### DISCUSSION TITLE
Pics disappear after EXIF-update

### DISCUSSION THREAD

**Discussion by kerkhc** [photoprism/photoprism#4123] [Q&A] [answered]
Hello you knowers,

I installed a PP Docker instance and start with some 20+ JPG-pics. Everything's fine so far...

Now I recognized that my pics have missing/wrong EXIF data (date-values, GPS-data).
So I updated the pics outside with EXIF-data using python-scripts. EXIF-data is shown correctly in e.g. IrfanView.

After copying the files over the existing ones on my photoprism "Pictures" folder, I reindexed all pics.

Finally only one (1 !) photo is shown in the UI.

How can I reindex files after changing EXIF-data ?
I assume that I missed some important step :/

**Comment by graciousgrey** [comment, marked as answer]
Hey :wave: 

Potential reasons for missing pictures are listed here:

https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

Did you set the same GPS and date-values for all or many of your files? In this case they might be stacked if you have stacking by place and time enabled.

https://docs.photoprism.app/user-guide/settings/library/#stacks
https://docs.photoprism.app/user-guide/organize/stacks/

Should the list not help I would recommend setting the log level to trace mode and watching the logs during indexing,

https://docs.photoprism.app/getting-started/troubleshooting/logs/#__tabbed_1_3

**Comment by kerkhc** [comment]
It was the stacking setting.
THANK YOU SO MUCH !
